### ISSUE TITLE
Resource Type Filter - 1.4.3 Contrast Minimum - Contrast between white-on-blue text of resource Type button is not sufficient

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9904]
As a user with visual impairments, I expect the contrast between text and background to be at least a ratio of 4.5:1, to ensure I can easily discern text.

**Linked Work Item:**
#9493 

**Expected Result:**
 A contrast ratio of 4.5:1 should be present between all text and background colours to ensure good visibility for people with vision impairments.

**Actual Result:**
Contrast between white text and blue background of the Resource Type button is only 2.89:1.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/520685a3-d290-4180-87f7-03429cbabdb9)

**Comment by imogenroberts** [comment]
RETEST: Contrast between the white text and blue (now navy blue) background is now consistent with WCAG standards

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/811c90c9-b570-4200-9906-04d5c8b43f2c)
